package/system/urandom-seed/Makefile

```diff
@@ -14,16 +12,10 @@ define Package/urandom-seed
   CATEGORY:=Base system
   DEPENDS:=+getrandom
   TITLE:=/etc/urandom.seed handling for OpenWrt
-  URL:=http://openwrt.org/
-endef
-
-define Build/Prepare
-	mkdir -p $(PKG_BUILD_DIR)
```

Comment:
I haven't checked the flags, but it seems that the default Build/Prepare does different/additional things?

Reply:
IIRC `Build/Prepare/Default` does nothing harmful, and `mkdir -p $(PKG_BUILD_DIR)` is executed somewhere in the general package initialization process. But I can leave it if you like.